src/lib/fragments/accordion.css

```diff
@@ -1,14 +1,5 @@
-.accordion-button:focus,
```

Comment:
AccordionButtonClassName uses the accordion-button class so will we have the correct background colour etc when the accordian header is focused if we remove this?

Reply:
The "jump to" chart button now has the same formatting as other secondary buttons on the dashboard - I spoke with Joe who said this was okay.

![image](https://github.com/communitiesuk/gov-uk-dash-react-components/assets/95618509/5bb49ba0-b697-433b-aee7-9ab43be065ee)